core/players-handbook/spells.xml

```diff
@@ -4152,7 +4306,21 @@
+			<p class="indent">When you cast the spell, you choose one of the following forms of imprisonment.</p>
+			<p class="indent"><strong><i>Burial. </i></strong>The target is entombed far beneath the earth in a sphere of magical force that is just large enough to contain the target. Nothing can pass through the sphere, nor can any creature teleport or use planar travel to get into or out of it.</p>
+			<p class="indent">The special component for this version of the spell is a small mithral orb.</p>
+			<p class="indent"><strong><i>Chaining. </i></strong>Heavy chains, firmly rooted in the ground, hold the target in place. The target is restrained until the spell ends, and it can't move or be moved by any means until then.</p>
+			<p class="indent">The special component for this version of the spell is a fine chain of precious metal.</p>
+			<p class="indent"><strong><i>Hedged Prison. </i></strong>The spell transports the target into a tiny demiplane that is warded against teleportation and planar travel. The demiplane can be a labyrinth, a cage, a tower, or any similar confined structure or area of your choice.</p>
+			<p class="indent">The special component for this version of the spell is a miniature representation of the prison made from jade.</p>
+			<p class="indent"><strong><i>Minimus Containment. </i></strong>The target shrinks to a height of 1 inch and is imprisoned inside a gemstone or similar object. Light can pass through the gemstone normally (allowing the target to see out and other creatures to see in), but nothing else can pass through, even by means of teleportation or planar travel. The gemstone can't be cut or broken while the spell remains in effect.</p>
+			<p class="indent">The special component for this version of the spell is a large, transparent gemstone, such as a corundum, diamond, or ruby.</p>
+			<p class="indent"><strong><i>Slumber. </i></strong>The target falls asleep and can't be awoken. The special component for this version of the spell consists of rare soporific herbs.</p>
+			<p></p>
+			<p class="indent"><strong><i>Ending the Spell. </i></strong>During the casting of the spell, in any of its versions, you can specify a condition that will cause the spell to end and release the target. The condition can be as specific or as elaborate as you choose, but the GM must agree that the condition is reasonable and has a likelihood of coming to pass. The conditions can be based on a creature's name, identity, or deity but otherwise must be based on observable actions or qualities and not based on intangibles such as level, class, or hit points.</p>
```

Comment:
```suggestion
			<p class="indent"><strong><i>Slumber. </i></strong>The target falls asleep and can't be awoken. The special component for this version of the spell consists of rare soporific herbs.</p>
			<p class="indent"><strong><i>Ending the Spell. </i></strong>During the casting of the spell, in any of its versions, you can specify a condition that will cause the spell to end and release the target. The condition can be as specific or as elaborate as you choose, but the GM must agree that the condition is reasonable and has a likelihood of coming to pass. The conditions can be based on a creature's name, identity, or deity but otherwise must be based on observable actions or qualities and not based on intangibles such as level, class, or hit points.</p>
```